script/glam/export_csv

```diff
@@ -8,7 +8,12 @@ src_project=${SRC_PROJECT:-glam-fenix-dev}
 # TODO: glam-dev-bespoke-nonprod-dataops-mozgcp-net
 dst_project=${DST_PROJECT:-$src_project}
 dataset=${DATASET:-glam_etl_dev}
-product=${PRODUCT:-org_mozilla_fenix}
+product=${PRODUCT}
```

Comment:
A shorthand uses `?` substituton
```suggestion
product=${PRODUCT?PRODUCT must be defined}
```

Reply:
Wow this is useful.  I had no idea

Reply:
I've written more bash than I've cared for. The Linux Documentation Project has a good reference on these parameter substitutions: https://www.tldp.org/LDP/abs/html/parameter-substitution.html